core/src/main/java/org/apache/calcite/avatica/MetaImpl.java

```diff
@@ -383,11 +405,29 @@ public MetaColumn(
       this.charOctetLength = charOctetLength;
       this.ordinalPosition = ordinalPosition;
       this.isNullable = isNullable;
+      this.isAutoincrement = isAutoincrement;
+      this.isGeneratedcolumn = isGeneratedcolumn;
     }
 
     public String getName() {
       return columnName;
     }
+
+    /** Returns a copy of this MetaColumn, overriding the value of {@code isAutoincrement}. */
+    @SuppressWarnings("unused") // called from Calcite
+    public MetaColumn withIsAutoIncrement(String isAutoincrement) {
```

Comment:
I would make it consistent either do AutoIncremeent or Autoincrement everywhere

Reply:
lol, I thought about this for too long. I'll change it for consistency.